### ISSUE TITLE
Browse Remote FTP server

### ISSUE THREAD

**Issue by PaoloFalomo** [lukasz-wronski/vscode-ftp-sync#80]
Hi Lukasz! 

Your extension is just **amazing** but it is essential to have a `browse remote feature` (like sublime text [sftp Package](https://wbond.net/sublime_packages/sftp/usage)).

In sublime text we have something like this (click to enlarge):
<a href="https://wbond.net/sublime_packages/img/sftp/remote_browsing_list.png" target="_blank">
<img src="https://wbond.net/sublime_packages/img/sftp/remote_browsing_list.png" width="80%"></a>

I think this feature will allow me to move from ST3 to VSCode! 

I am a developer and i am 100% available to contribute 😃

**Comment by ivansf** [comment]
Actually, Atom editor with https://atom.io/packages/remote-ftp works better for me than sublimetext

**Comment by PaoloFalomo** [comment]
@ivansf Yes! There are so much good editors... Visual Studio Code is one of best but he's "young" 😄

**Comment by marcoczen** [comment]
How dow we actually use this pluin ? Is there a "setup" icon ?
I have installed the extension but cannot get it setup ?
